docs/en/pid-issuing.rst

```diff
@@ -0,0 +1,598 @@
+    4. **End-user authentication**: the PID Provider authenticates the End-User using a traditional eIDAS high LoA SAML2/OIDC authentication acting as an IAM Proxy. 
+    5. **PID issuance**: finally, after the End-User authentication with LoA High and consent, and the check of the Wallet Instance by means of the Wallet Attestation and the Trust Chain related to the Wallet Provider as the provider of a compliant Wallet Solution, the PID Provider responds with a PID, bound to key material as requested.
+
+The Wallet Instance Setup phase is described in Section [...]. In the following Section the steps from 1 to 5 are further expanded into more technical detailed steps. 
+
+Detailed Flow
+-------------
+
+The PID Issuing phase is based on the **Authorization Code Flow** with **Pushed Authorization Requests** (PAR) [:rfc:`9126`] and **PKCE** (Proof Key for Code Exchange, :rfc:`7636`) as recommended in `[OIDC4VCI. Draft 13. Section 3.4] <https://openid.bitbucket.io/connect/openid-4-verifiable-credential-issuance-1_0.html#section-3.4>`_. A *Wallet Initiated Flow* is considered and the User receives the PID directly in response to the Credential Request (*Immediate Flow*) on the same device as the device the Wallet Instance resides on (*Same-device*). 
+
+.. _fig_Low-Level-Flow-EUDIW-PID-Issuing:
+.. figure:: ../../images/Low-Level-Flow-EUDIW-PID-Issuing.svg
+    :figwidth: 100%
+    :align: center
+    :target: https://www.plantuml.com/plantuml/png/hLHHQoCt47xNhnXo1xt0Xhu7V5noSI-cePWqnxqK5dbtx1sobhRIA3xtrtSaUHUjcnuf5FpWrSmttyptQVJ9itOSzhtIWQq9-mqwfGOveuO6RHWkldfq5w0z1AKszdi--Cc3tqlMq5c35_3kWNOTt_O1vzXRPmyJNCzZttJV8yFAI0kcmPHpK6XQ41RrQkxQsHTgJ_KsPpdhrTqyWSuIS7LtypkKQV27UNB1CxRmG9uJYfLIYqLA8-y3cHt0Og7K0Qu-GhW6ks5D1eusk9Ya0bn9u70DH_HAEMCPmRv8xKF5yExR_UffgKuO3RsmWDt2o5es4qTFSE6w-YXCZJLRSdlW3i5ZZmsJDOAUerGGVg6tpsVi9Awiljm_VNbSBdzJPsf3qs7pxAMsvaWmbuKyHEM4AyL-GJdKYLsRTbI0sCCUMRUQTTaxH3b0sdUeMGh_-dafexG8qXPj2TqfXoHdVVC8Z_XdGC_mmQ4VhF5Oyt702fgUq71DRPNWTTF54SmEpxzhwQApakPofGcglKSNsougbiUBzrcLKPGRPe6dGTzVGgacLPkPcZErpgNCHT-Yve_hUDePpS6X8Cl2HPiG3KbElEYUMk8Z55OeodTp4ZhhAqfdeUhWg0ByFf18NvEv5CWXGOA-XfiWgZhwAnDDBXMs56KkLT09IVyPgEP2Xt5fWfTlBRK4HKqkTY3k8E_nFUswD7qJPp9Sp4sd6U2ibqA1U8CgY2hYrFhXZVd57T-XYLuYtAtj6fxn-A-nmLbnSMlTVDFOFgC1QNsm93CKg_4aaKczF7XHg5HpvxJXyOhRTjoFSTt-btM8dhNai1bDepncxhDKIZSDUb-dmQeyN-OhBffmryOAnVIdnYdughvRYNLHu2HX8ysWOTBzFz-Rh8_y1Yk3UHzz_91PidL5_e_TANacYmg7Fa_HLsmgPOZNT6xUXAnRt9AXg74L-VUQgvnrMQx5-9RVFdnTgaVSaMSv5RU0EopQl7BOQyOM59bSJGgdkV9jo6lqiyCNs-GOb7kJfIadAumzTLUmlTd6AEhqG8aOQP62pozUaUVPEYpT_IJDXNt_Dm00
+
+    PID Issuing - Detailed flow
+
+**Steps 1-4:** The User has selected a PID Provider, and the Wallet Instance obtains the metadata for the selected PID Provider.
+
+.. note::
+
+    **Federation Check:** The Wallet Instance needs to check if the PID Provider is part of Federation and then it can consume its Metadata.
```

Comment:
let's say how -> Federation entity Discovery and the build of the federation trust chain related to the credential instance